File: backend/ProductSale.Repository/Entities/StoreLocation.cs
using System;
using System.Collections.Generic;

namespace ProductSale.Repository.Entities;

public partial class StoreLocation
{
    public int LocationId { get; set; }

    public decimal Latitude { get; set; }

    public decimal Longitude { get; set; }

    public string Address { get; set; } = null!;
}

File: mobile/app/src/main/java/com/prm392/assignment/productsale/model/products/StoreLocation.java
package com.prm392.assignment.productsale.model.products;

import com.google.gson.annotations.SerializedName;

import lombok.Getter;

@Getter
public class StoreLocation {
    @SerializedName(value = "locationId")
    private String locationId;
    @SerializedName(value = "latitude")
    private Double latitude;
    @SerializedName(value = "longitude")
    private Double longitude;
    @SerializedName(value = "address")
    private String address;
}
